workspaces/Makefile

```diff
@@ -32,7 +32,7 @@ nitro: shared
 	$(MAKE) -C nitro-runtime
 
 icecap: shared
-	$(MAKE) -C icecap-runtime build
+	$(MAKE) -C icecap-runtime build PROFILE=release
 	$(MAKE) -C icecap-host
 
```

Comment:
I have concerns about forcing the value of the `PROFILE` (it will take precedence over the value set by user), and also having possibly different values for `icecap-runtime` and `icecap-host`.

Reply:
The issue I am trying to solve is that anything other than PROFILE=release does not work (and I wasted 1.5 days of work figuring this out). How would you propose we solve this?

Reply:
We should fix the `dev` profile settings in `workspace/icecap-runtime/Cargo.toml` so that it works.

Reply:
done